examples/EvaluationDataToApplicationInsights/Program.cs

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+using Microsoft.FeatureManagement.Telemetry.ApplicationInsights;
+using EvaluationDataToApplicationInsights.Telemetry;
+using Microsoft.FeatureManagement;
+using EvaluationDataToApplicationInsights;
+using Microsoft.ApplicationInsights.Extensibility;
+
+var builder = WebApplication.CreateBuilder(args);
+
+//
+// Use Azure App Configuration as an application configuration provider
+//
+builder
+    .Configuration
+    .AddAzureAppConfiguration((options) =>
```

Comment:
Is it necessary to use App Configuration here? All feature flag configurations are listed in the appsettings.json. 
README doesn't mention anything about setting connection string for App Config.
Perhaps change to
```C#
if (!string.IsNullOrEmpty(builder.Configuration["AzureAppConfigurationConnectionString"]))
{
    builder.AddAzureAppConfiguration(options =>
    {
        options.Connect(settings["AppConfiguration:ConnectionString"]);
    });
}
```

Reply:
No actually it should be removed. Thanks.

Reply:
Removed.